lib/txt/news

```diff
@@ -1,3 +1,7 @@
+09-15-20 : Stat updates
+            - Spells and skills now have a modification stat that affects 
+              damage and duration of buffs/debuffs. See helpfiles
```

Comment:
Are the helpfiles updated somewhere else?

Reply:
In code/code/cmd/cmd_help.cc I added "modifier stat" and it pulls from the discarray. 

Looks like this:

<img width="625" alt="Screen Shot 2020-09-15 at 11 15 42 AM" src="https://user-images.githubusercontent.com/1966964/93248755-2728a280-f745-11ea-9add-ec464ec80d11.png">

Or for skills where we used STAT_EXT it will say "none"

<img width="510" alt="Screen Shot 2020-09-15 at 11 21 45 AM" src="https://user-images.githubusercontent.com/1966964/93249138-ad44e900-f745-11ea-8c07-2f2c4aa29ab3.png">

Reply:
I think it would be better if it told you exactly how the modifier stat affects the skill/spell but I think for most skill it should be pretty obvious.